### DISCUSSION TITLE
Qiitaの開発ロードマップを正式に公開します！

### DISCUSSION THREAD

**Discussion by tanaka-asami415** [increments/qiita-discussions#449] [アナウンス]
[Qiitaの開発ロードマップ](https://github.com/orgs/increments/projects/8/views/1)を正式に公開します！

開発ロードマップとは、運営が今何に力を入れているのか、どういったものを将来的に考えているのか、
過去にどんなプロジェクトを実施してきたのか、そういった情報をユーザーのみなさんへオープンにするための取り組みです。

今までも試験的に公開していましたが (https://github.com/increments/qiita-discussions/discussions/175) この度、本格的に公開することにいたしました。

今後のロードマップでは以下の内容を確認いただけます。

- 現在優先的に取り組んでいるものや進捗
- 未着手のもの
- リリースや不具合の解消が完了したもの

これからは、ダークモードのような大きなプロジェクト単位ではなく、不具合解消等の小さな取り組みも公開し、
運営が今現在取り組んでいる状態を、よりオープンかつタイムリーに
みなさんにお伝えしていきたいと考えていますので、ぜひご活用いただけますと幸いです！✨

**Comment by tsubolabo** [comment]


**Reply by tanaka-asami415** [reply]
@tsubolabo 
回答が遅くなりました🙏
プロジェクト単位のものは、まずは直近で取り組む予定の内容を掲載しておりますが、
今後どういったタイミングで載せていくかは、運用をする中で決めていきたいと考えています。

不具合解消等の細かな内容については、着手のタイミングで順次追加してまいります！
（今後、ロードマップの運用を継続してみて、掲載ルールが変わることはあるかもしれません。）

また、ヘルプページに関しては開発とは少し異なるため、ロードマップに掲載の予定はありません。

> 「参考にさせていただきます」はほぼ「没扱い」と認識しておりますが

ご要望いただいてから対応ができておらず、そう見えていたら申し訳ありませんが、
表現の違いによって実施するか否かの区別がある訳ではございません🙇

**Comment by tanaka-asami415** [comment]
[質問者・回答者以外も回答に対してコメントができるようにする](https://github.com/increments/qiita-discussions/issues/447)
[回答コメントを通報出来るようにする](https://github.com/increments/qiita-discussions/issues/451) がリリースされました！🙌✨

[リリースノート](https://qiita.com/release-notes/2023-03-24)

引き続き開発の進捗状況をロードマップで随時公開していきますので、ぜひご確認ください✨
